backend/src/apps/github/models/issue.py

```
@@ -173,19 +173,28 @@ def generate_summary(self, open_ai: OpenAi | None = None, max_tokens: int = 500)
-        open_ai = open_ai or OpenAi()
-        open_ai.set_input(f"{self.title}\r\n{self.body}")
-        open_ai.set_max_tokens(max_tokens).set_prompt(prompt)
-        self.summary = open_ai.complete() or ""
+        try:
+            open_ai = open_ai or OpenAi()
+            open_ai.set_input(f"{self.title}\r\n{self.body}")
+            open_ai.set_max_tokens(max_tokens).set_prompt(prompt)
+
+            summary = open_ai.complete()
+            self.summary = summary.strip() if summary and summary.strip() else fallback_summary
+        except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)